### DISCUSSION TITLE
how to update wikijs.

### DISCUSSION THREAD

**Discussion by John357John** [requarks/wiki#7444] [Help / Questions]
i'm running in the 2.5.303, and my wiki start in docker, how can i update it to 2.5.305 without data losing.
I try new docker image start, but it will begin the setup.js.

**Comment by BPowell76** [comment]
What have you tried?

I don't use Docker for the wiki instance, but the page contents are stored in a database so you won't lose it unless you lose the database. The docker system should have a wiki update monitor. Have you tried these steps?

- [Docker Upgrade in Place](https://docs.requarks.io/install/upgrade)

The manual update process is simple, just delete the existing files and paste the new web app files.

**Reply by John357John** [reply]
Thx for you help.
My wiki worked offline, so i have to update docker image manually. i dont want to lose any data.
My step is :
I deleted old image and loaded new image by manually. Then i open the wiki, it ask me to install wiki, but the database have the account so i cannot finish the installation.
I will try to update the wiki by the guide again.

**Reply by BPowell76** [reply]
@John357John Did the update process work out okay?

**Reply by John357John** [reply]
Yeah, delete current old wiki image, upload new wiki image and start, it worked
